server/migrations/20230804034846-migrate-emoji-in-document-title.js

```diff
@@ -0,0 +1,67 @@
+"use strict";
+
+const path = require("path");
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    if (process.env.DEPLOYMENT === "hosted") {
+      return;
+    }
+
+    if (process.env.NODE_ENV === "test") {
+      return;
+    }
+
+    const rootDir = "build";
+
+    const { Document } = require(path.join(
+      process.cwd(),
+      rootDir,
+      "server/models"
+    ));
+    const { default: parseTitle } = require(path.join(
+      process.cwd(),
+      rootDir,
+      "shared/utils/parseTitle"
+    ));
+
+    await queryInterface.sequelize.transaction(async (transaction) =>
+      Document.unscoped().findAllInBatches(
```

Comment:
Hmm...you seem to be running the migration using cli. Tried with `make up` or `yarn dev`? That way all models are initialized before any migration runs.

Reply:
Yea, running `yarn db:migrate`, this has to work in production too so that needs to work independently 😬

Reply:
Urgh! That's gonna be a pain without the model being available :grimacing:

Reply:
It might be that we just shell out to the script from the migration? It would be a first… but agreed that we need to use the models here and would avoid having the logic twie

Reply:
Yeah, we can do for this case now. But going forward, I believe better to move to `umzug` completely and avoid sequelize-cli since sequelize-cli anyway uses `umzug` underneath.

Reply:
By the way, they seem to have rolled out cli as of v3 https://github.com/sequelize/umzug#cli

Reply:
Ok, happy to tackle as a followup

Reply:
> It might be that we just shell out to the script from the migration? It would be a first… but agreed that we need to use the models here and would avoid having the logic twice

It seems like you removed the migration entirely? I suggested to shell out, e.g use `exec` to call the script from the migration, that way we can assure it is run successfully still.

Reply:
Sorry, added it back :+1: